src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentCostPerDay = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const daysForMinDisc = 3;
+  const daysForMaxDisc = 7;
+  const rentCost = days * rentCostPerDay;
+  const rentCostWithMinDisc = rentCost - minDiscount;
+  const rentCostWithMaxDisc = rentCost - maxDiscount;
+
+  if (days < daysForMinDisc) {
+    return rentCost;
+  } else if (days < daysForMaxDisc) {
+    return rentCostWithMinDisc;
+  } else {
+    return rentCostWithMaxDisc;
+  }
```

Comment:
```suggestion
  if (days < daysForMinDisc) {
    return rentCost;
  }
  
  if (days < daysForMaxDisc) {
    return rentCostWithMinDisc;
  }
  
  return rentCostWithMaxDisc;
```
Если у тебя в каждом if return - else можно не писать

Reply:
Понял, спасибо.

Reply:
add solution for actual task version